sdk/monitor/monitor-query/samples-dev/logsQuery.ts

```diff
@@ -48,8 +48,13 @@ export async function main() {
     return;
   }
 
+  const executionTime =
+    result.statistics && result.statistics.query && result.statistics.query.executionTime;
+
   console.log(
-    `Results for query '${kustoQuery}', execution time: ${result.statistics?.query?.executionTime}`
+    `Results for query '${kustoQuery}', execution time: ${
+      executionTime == null ? "unknown" : executionTime
```

Comment:
`executionTime || "unknown"` might be nicer, but its just a stylistic preference so feel free to disagree!

Reply:
I have actually seen it sometimes be zero :)

I also really just wish I could use ??. :)